akvo/rest/views/project_editor.py

```diff
@@ -810,12 +810,20 @@ def project_editor_import_results(request, project_pk=None):
     project = Project.objects.get(pk=project_pk)
     user = request.user
 
-    if not (user.is_superuser or user.can_import_results()):
+    if not (user.is_superuser or
+            user.can_import_results() and user.has_perm('rsr.change_project', project)):
```

Comment:
`user.can_import_results` seems to be doing something that doesn't make a lot of sense - verifying if the user has at least one `Admin` or `M&E manager` employment.  Just checking for `rsr.change_project` permissions seems like a much better way of doing this, but also gives the permission to Project Editors. It may be worth adding another permission `rsr.can_import_results` instead, and use that.